Dockerfile

```diff
@@ -1,12 +1,16 @@
 ####################################################################
-FROM ubuntu:22.04 as builder 
+FROM ubuntu:23.04 as builder
 MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
 
 ARG CONCURRENCY
 
 # set paths
-ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
-ENV LD_LIBRARY_PATH /usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
+ENV LD_LIBRARY_PATH=/usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
+# python stuff
+ENV VIRTUAL_ENV=/usr/local/venv
+RUN export DEBIAN_FRONTEND=noninteractive && apt update && apt install -y sudo python3-venv && python3 -m venv ${VIRTUAL_ENV}
```

Comment:
hm, a venv is quite heavy.. on my local it weighs 26 mb.

Reply:
i had the same feeling about the size of the sqlite db update in the phonetics pr